ArcGISRuntimeSDKQt_CppSamples/Geometry/ClipGeometry/main.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <QGuiApplication>
+#include <QQuickView>
+#include <QCommandLineParser>
+#include <QDir>
+#include <QQmlEngine>
+
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#endif
+
+#include "ClipGeometry.h"
+
+#define STRINGIZE(x) #x
+#define QUOTE(x) STRINGIZE(x)
+
+int main(int argc, char *argv[])
+{
+
+  QGuiApplication app(argc, argv);
+  app.setApplicationName(QStringLiteral("%{Clip geometry} - C++"));
```

Comment:
`Clip Geometry - C++`

Reply:
I think for this one the lowercase `g` is something Lucas told me to do, since I usually capitalize each word in a sample title

Reply:
ok, that's fine. Thanks @acmorris.